### ISSUE TITLE
Can't install the tensorflow version

### ISSUE THREAD

**Issue by satroz** [kundajelab/deeplift#16]
Hello, I know it's still in alpha, but I tried to install the tensorflow version as per instructions
and I got the following error after running the pip install:

error: package directory 'deeplift/backend' does not exist

And yes, I don't see the "backend" folder, though it is used by util.py

**Comment by AvantiShri** [comment]
Hi, thanks a lot for flagging this. I just pushed a fix removed the reference to deeplift.backend from setup.py and from util.py. I also updated the code to work with tensorflow 1.0.1. I ran the unit tests and they passed except for minor numerical inaccuracy in the batchnorm case (results are within E-3 instead of E-5). This may be due to a change in the batchnorm implementation between 0.12rc and 1.0.1. Let me know if this E-2 discrepancy is a problem for you and if so, I can look into it. And please do let me know if you run into any other issues!
